src/paperless_mail/views.py

```diff
@@ -24,3 +33,41 @@ class MailRuleViewSet(ModelViewSet, PassUserMixin):
     serializer_class = MailRuleSerializer
     pagination_class = StandardPagination
     permission_classes = (IsAuthenticated,)
+
+
+class MailAccountTestView(GenericAPIView):
+
+    permission_classes = (IsAuthenticated,)
+    serializer_class = MailAccountSerializer
+
+    def post(self, request, *args, **kwargs):
+        logger = logging.getLogger("paperless_mail")
+        request.data["name"] = datetime.datetime.now().isoformat()
```

Comment:
I don't see this used by the frontend, is it a leftover?

Reply:
You mean the `post` itself? That’s https://github.com/paperless-ngx/paperless-ngx/blob/5012c0c97c5f7bcf59f0485fad513d5faf2df618/src-ui/src/app/services/rest/mail-account.service.ts#L52

or do you just mean the name part? That just prevents the serializer complaining since we’re not going to save this object

lmk if I misunderstood your question

Reply:
Yeah, I was wondering why the write to the request existed. But if it silences some warning, that's alright with me